src/diffusers/loaders/unet.py

```diff
@@ -561,10 +562,121 @@ def _unfuse_lora_apply(self, module):
             if isinstance(module, BaseTunerLayer):
                 module.unmerge()
 
+    def _expand_lora_scales_dict(
```

Comment:
should we make a `unet_loader_utils` file? move all these added functions there? cc @sayakpaul

Reply:
Works for me.

Reply:
@yiyixuxu There's only 1 added function (`_expand_lora_scales_dict`) in `loaders/unet.py`. Are you sure we should create a new file for that?

Or are there other functions you're referring to?

Reply:
see this comment here :) https://github.com/huggingface/diffusers/pull/7352/files#r1535961194